### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Update dependencies

#### Ticket Link
NONE

### CODE REVIEW

**Comment by larkox** [comment]
@enahum I have no permissions to add reviewers to this PR

File: example/ios/ReactNativeTurboLogExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -286,6 +262,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = V9GZ4T2337;
```
Comment (enahum) [inline example/ios/ReactNativeTurboLogExample.xcodeproj/project.pbxproj]: remove

File: example/ios/ReactNativeTurboLogExample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -312,6 +289,7 @@
...
```
Comment (enahum) [inline example/ios/ReactNativeTurboLogExample.xcodeproj/project.pbxproj]: remove

File: example/ios/ReactNativeTurboLogExample.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -379,7 +357,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 12.4;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.4;
```
Comment (enahum) [inline example/ios/ReactNativeTurboLogExample.xcodeproj/project.pbxproj:358]: this is fine but for the app it needs to go in release notes

**Review by enahum — CHANGES_REQUESTED** [review]

File: example/ios/ReactNativeTurboLogExample.xcodeproj/project.pbxproj (hunk 1 above)
Comment (larkox) [reply example/ios/ReactNativeTurboLogExample.xcodeproj/project.pbxproj]: I thought I removed it :facepalm:

**Review by enahum — APPROVED** [review]

**Comment by enahum** [comment]
I'll publish it shortly

**Comment by enahum** [comment]
@larkox published

**Pull request merged**